Traceback (most recent call last):
  File "/home/z0272571a@CAMPUS.CSIC.ES/miniconda3/envs/pyCATHY_doc/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/z0272571a@CAMPUS.CSIC.ES/miniconda3/envs/pyCATHY_doc/lib/python3.10/site-packages/nbclient/client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/z0272571a@CAMPUS.CSIC.ES/miniconda3/envs/pyCATHY_doc/lib/python3.10/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
  File "/home/z0272571a@CAMPUS.CSIC.ES/miniconda3/envs/pyCATHY_doc/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/home/z0272571a@CAMPUS.CSIC.ES/miniconda3/envs/pyCATHY_doc/lib/python3.10/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/home/z0272571a@CAMPUS.CSIC.ES/miniconda3/envs/pyCATHY_doc/lib/python3.10/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/z0272571a@CAMPUS.CSIC.ES/miniconda3/envs/pyCATHY_doc/lib/python3.10/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
SPP, FP = simu.read_inputs('soil', MAXVEG=8)
FP
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
Cell [0;32mIn[7], line 1[0m
[0;32m----> 1[0m SPP, FP [38;5;241m=[39m [43msimu[49m[38;5;241;43m.[39;49m[43mread_inputs[49m[43m([49m[38;5;124;43m'[39;49m[38;5;124;43msoil[39;49m[38;5;124;43m'[39;49m[43m,[49m[43m [49m[43mMAXVEG[49m[38;5;241;43m=[39;49m[38;5;241;43m8[39;49m[43m)[49m
[1;32m      2[0m FP

File [0;32m~/miniconda3/envs/pyCATHY_doc/lib/python3.10/site-packages/pyCATHY-0.1.1-py3.10.egg/pyCATHY/cathy_tools.py:3858[0m, in [0;36mCATHY.read_inputs[0;34m(self, filename, **kwargs)[0m
[1;32m   3855[0m [38;5;28;01melse[39;00m:
[1;32m   3856[0m     MAXVEG [38;5;241m=[39m [38;5;28mself[39m[38;5;241m.[39mMAXVEG
[0;32m-> 3858[0m df [38;5;241m=[39m [43min_CT[49m[38;5;241;43m.[39;49m[43mread_soil[49m[43m([49m
[1;32m   3859[0m [43m    [49m[43mos[49m[38;5;241;43m.[39;49m[43mpath[49m[38;5;241;43m.[39;49m[43mjoin[49m[43m([49m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mworkdir[49m[43m,[49m[43m [49m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mproject_name[49m[43m,[49m[43m [49m[38;5;124;43m"[39;49m[38;5;124;43minput[39;49m[38;5;124;43m"[39;49m[43m,[49m[43m [49m[43mfilename[49m[43m)[49m[43m,[49m
[1;32m   3860[0m [43m    [49m[43mdem_parm[49m[43m,[49m
[1;32m   3861[0m [43m    [49m[43mMAXVEG[49m[38;5;241;43m=[39;49m[43mMAXVEG[49m[43m,[49m
[1;32m   3862[0m [43m[49m[43m)[49m
[1;32m   3863[0m [38;5;66;03m# df[0][39;00m
[1;32m   3864[0m [38;5;28;01mreturn[39;00m df

File [0;32m~/miniconda3/envs/pyCATHY_doc/lib/python3.10/site-packages/pyCATHY-0.1.1-py3.10.egg/pyCATHY/importers/cathy_inputs.py:396[0m, in [0;36mread_soil[0;34m(soilfile, dem_parm, MAXVEG)[0m
[1;32m    394[0m FP_soil_mat [38;5;241m=[39m []
[1;32m    395[0m [38;5;28;01mfor[39;00m fp [38;5;129;01min[39;00m FP_soil:
[0;32m--> 396[0m     FP_soil_mat[38;5;241m.[39mappend([[38;5;28mfloat[39m(fpi) [38;5;28;01mfor[39;00m fpi [38;5;129;01min[39;00m fp [38;5;28;01mif[39;00m [38;5;28mlen[39m(fpi)[38;5;241m>[39m[38;5;241m0[39m])
[1;32m    398[0m FP_soil_mat [38;5;241m=[39m np[38;5;241m.[39marray(FP_soil_mat)
[1;32m    399[0m df_FP [38;5;241m=[39m pd[38;5;241m.[39mDataFrame(FP_soil_mat)

File [0;32m~/miniconda3/envs/pyCATHY_doc/lib/python3.10/site-packages/pyCATHY-0.1.1-py3.10.egg/pyCATHY/importers/cathy_inputs.py:396[0m, in [0;36m<listcomp>[0;34m(.0)[0m
[1;32m    394[0m FP_soil_mat [38;5;241m=[39m []
[1;32m    395[0m [38;5;28;01mfor[39;00m fp [38;5;129;01min[39;00m FP_soil:
[0;32m--> 396[0m     FP_soil_mat[38;5;241m.[39mappend([[38;5;28;43mfloat[39;49m[43m([49m[43mfpi[49m[43m)[49m [38;5;28;01mfor[39;00m fpi [38;5;129;01min[39;00m fp [38;5;28;01mif[39;00m [38;5;28mlen[39m(fpi)[38;5;241m>[39m[38;5;241m0[39m])
[1;32m    398[0m FP_soil_mat [38;5;241m=[39m np[38;5;241m.[39marray(FP_soil_mat)
[1;32m    399[0m df_FP [38;5;241m=[39m pd[38;5;241m.[39mDataFrame(FP_soil_mat)

[0;31mValueError[0m: could not convert string to float: 'IVGHU\n'

